### ISSUE TITLE
[change] Support full range of official WiFi 5GHz channels

### ISSUE THREAD

**Issue by lpalgarvio** [openwisp/netjsonconfig#233]
The current channel list is incomplete for 5Ghz / 802.11ac.
netjsonconfig currently only accepts channels 36-64 and 100-140.
Source: https://github.com/openwisp/netjsonconfig/blob/master/netjsonconfig/channels.py

As per specs, the channels are divided in groups of bands:
- U-NII-1: 36-48@20MHz, 38-46@40MHz, 42@80MHz (encompasses 32-50)
- U-NII-2A: 52-64@20MHz, 54-62@40MHz, 58@80MHz (encompasses 50-68)
- U-NII-2C: 100-144@20MHz, 102-142@40MHz, 106-138@80MHz (encompasses 96-144)
- U-NII-3: 149-165@20MHz, 151-159@40MHz, 155@80MHz (encompasses 138-171)
- U-NII-4: 165-177@20MHz, 167-175@40MHz, 171@80MHz (encompasses 163-177)
Source1: https://en.wikipedia.org/wiki/List_of_WLAN_channels
Source2: https://www.ekahau.com/blog/channel-planning-best-practices-for-better-wi-fi/

As of OpenWrt 20.02.03, current accepted channels are:
- 36 to 64, increments of 4 (U-NII-1 + U-NII-2A)
- 100 to 144, increments of 4 (U-NII-2C)
- 149 to 173, increments of 4 (U-NII-3)
Using country PT (to 165 using country US) on a mac20811 device.

So in short, netjsonconfig has complete U-NII-1 and U-NII-2A, and incomplete U-NII-2C (up to 140 but not 144).
It is missing U-NII-3.
Might be missing U-NII-4 if OpenWrt/hostapd already accepts it.
I could not find confirmation on this last one yet, but its already overlapping a bit of it at least on my device...

So with this i request a that the missing channels be added.
- 100 to 144, increments of 4 (U-NII-2C), instead of up to 140
- 149 to 177, increments of 4 (U-NII-3 + U-NII-4), entirely missing

![image](https://user-images.githubusercontent.com/903711/171325900-634b5571-3a27-4d30-a7ef-43b13308d035.png)
![image](https://user-images.githubusercontent.com/903711/171329885-b2b7a353-0c40-498b-af2e-6fc0376f6b19.png)
